s4e-backend/src/main/java/pl/cyfronet/s4e/controller/SceneController.java

```diff
@@ -40,19 +41,22 @@
 
     @Operation(summary = "View a list of scenes")
     @ApiResponses({
-            @ApiResponse(responseCode = "200", description = "Successfully retrieved list")
+            @ApiResponse(responseCode = "200", description = "Successfully retrieved list"),
+            @ApiResponse(responseCode = "404", description = "Product not found", content = @Content)
     })
     @GetMapping("/products/{id}/scenes")
```

Comment:
Should be this endpoint visible to anyone?

Reply:
Yes it should, it's used by front-end to populate timeline